### DISCUSSION TITLE
Why not maintained anymore

### DISCUSSION THREAD

**Discussion by m-birke** [greater-than/PipeLayer#20] [General]
Hi @greater-than @aebenson 

I am also developing a very similar framework and it is a funny coincident that I also named it pipelayer. I would be interested why you dont maintain this project anymore. Are there other frameworks which make this obsolete or was it just not of interest anymore?

Thanks and KR

**Comment by greater-than** [comment]
Hi,
And thanks for reaching out. Funny, I don't have any feature updates planned, but I was just thinking about testing the library against the newer versions of Python and creating a new release. If you are interested joining forces and contributing to this repo, let me know.

Andrew

**Reply by greater-than** [reply]
Although, I do want to convert the PowerShell scripts to bash and/or make files.

**Comment by greater-than** [comment]
I'm working on a 0.8.0 release that is compatible with Python 3.9 - 3.12

**Comment by m-birke** [comment]
Hi, cool, thanks for the info :)
About joining forces: unfortunately right now not possible for me but when I see an opportunity I will reach out to you, sure.
